BibTeX.js

```diff
@@ -976,7 +976,11 @@ function beginRecord(type, closeChar) {
 					}
 					delete item.backupLocation;
 				}
-				
+				if (item.itemID) {
+					item._extraFields.push({ field: "citation key", value: item.itemID });
+					//item.citationKey = item.itemID;
```

Comment:
I don't know if this is a bug that maybe someone wants to look into but `.citationKey` doesn't seem to get saved to the database, or even exported to the output in the tests, which is odd because `doExport()` references it.